.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: bug
```

Comment:
Is it worth giving a list of possible labels? e.g. , 'bug', 'requested_expansion', 'question'

Reply:
Good idea:
```suggestion
labels: bug [e.g.: 'bug', 'enhancement', 'question']
```